healthtools_ke_api/views/nhif_outpatient_cs.py

```diff
@@ -0,0 +1,63 @@
+from flask import Blueprint, request, jsonify, make_response, json, current_app
+from bs4 import BeautifulSoup
+from elastic_search import Elastic
+
+from healthtools_ke_api.analytics import track_event
+
+import requests
+
+nhif_outpatientcs_api = Blueprint('nhif_outpatientcs_api', __name__)
```

Comment:
Let's call this `nhif_outpatient_cs_api`.

Reply:
changed to ```nhif_outpatient_cs_api```